naga/tests/in/unconsumed_vertex_outputs_vert.param.ron

```diff
@@ -1,2 +1,7 @@
 (
+	targets: "HLSL",
+	fragment_module: Some(FragmentModule(
+		path: "unconsumed_vertex_outputs_frag.wgsl",
+		entry_point: "fs_main"
+	)),
```

Comment:
thought: I'm honestly not sure that it's worth subsuming the `unconsumed_*` snapshots' weirdness with a separate fragment output into this. I'm willing to experiment, though, so I won't block on this.

Reply:
It doesn't feel that weird of a thing imo, and we might encounter other tests that require it. This is why I avoided making it a hlsl specific option